rest/model/image.go

```diff
@@ -0,0 +1,98 @@
+package model
+
+import (
+	"time"
+
+	"github.com/evergreen-ci/evergreen/thirdparty"
+	"github.com/evergreen-ci/utility"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIPackageOpts is the model to be returned by the API whenever package opts are fetched
+
+type APIPackageOpts struct {
+	Name    *string `json:"name"`
+	Manager *string `json:"manager"`
+	Limit   int     `json:"limit"`
+	Page    int     `json:"page"`
+}
+
+// BuildFromService converts from service level thirdparty.PackageFilterOptions to an APIPackageOpts
+func (apiPackageOpts *APIPackageOpts) BuildFromService(pkg thirdparty.PackageFilterOptions) {
+	apiPackageOpts.Name = utility.ToStringPtr(pkg.Name)
+	apiPackageOpts.Manager = utility.ToStringPtr(pkg.Manager)
+	apiPackageOpts.Limit = pkg.Limit
+	apiPackageOpts.Page = pkg.Page
+}
+
+// ToService returns a service layer package options using the data from APIPackageOpts
+func (apiPackageOpts *APIPackageOpts) ToService() *thirdparty.PackageFilterOptions {
+	pkgOpts := thirdparty.PackageFilterOptions{}
+	pkgOpts.Name = utility.FromStringPtr(apiPackageOpts.Name)
+	pkgOpts.Manager = utility.FromStringPtr(apiPackageOpts.Manager)
+	pkgOpts.Limit = apiPackageOpts.Limit
+	pkgOpts.Page = apiPackageOpts.Page
+	return &pkgOpts
+}
+
+////////////////////////////////////////////////////////////////////////////////
+//
+// APIPackage is the model to be returned by the API whenever packages are fetched
+
+type APIPackage struct {
+	Name    *string `json:"name"`
+	Manager *string `json:"manager"`
+	Version *string `json:"version"`
+}
+
+// BuildFromService converts from service level thirdparty.Package to an APIPackage
+func (apiPackage *APIPackage) BuildFromService(pkg thirdparty.Package) {
+	apiPackage.Name = utility.ToStringPtr(pkg.Name)
+	apiPackage.Manager = utility.ToStringPtr(pkg.Manager)
+	apiPackage.Version = utility.ToStringPtr(pkg.Version)
+}
+
+// ToService returns a service layer package using the data from APIPackage
```

Comment:
nit. comments should end with proper punctuation (also applies to comments below this)